EssayAnalyzer.Api/appsettings.json

```diff
@@ -5,9 +5,9 @@
       "Microsoft.AspNetCore": "Warning"
     }
   },
-  "OpenAiKey": "CHAT_GPT_API_KEY",
+  "OpenAiKey": "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",
```

Comment:
Do not push API key to GitHub Stash appsetting.jon

Reply:
I changed It